data/brands/amenity/fast_food.json

```diff
@@ -1703,7 +1703,7 @@
       "displayName": "Domino's",
       "id": "dominos-658eea",
       "locationSet": {"include": ["001"]},
-      "matchNames": ["dominos pizza"],
+      "matchNames": ["dominos pizza", "доміно"],
```

Comment:
In my opinion and other countries/regions' practice, you can create a new entry with locationSer=ua, and set tags like this:

```json
"tags": {
        "amenity": "fast_food",
        "brand": "Domino's",
        "brand:en": "Domino's",
        "brand:uk": "          ",
        "brand:wikidata": "Q839466",
        "cuisine": "pizza",
        "name": "Domino's",
        "name:en": "Domino's",
        "name:uk": "           ",
        "takeaway": "yes"
      }
```

Reply:
I edit my suggestion above, If you want to catch mapper's input `доміно`, I think you don't need to change global one, because other places' people don't call Domino as `доміно`